authorize/evaluator/opa/policy/headers.rego

```diff
@@ -204,43 +177,49 @@ kubernetes_headers = h {
 		["Impersonate-User", jwt_payload_email],
 		["Impersonate-Group", get_header_string_value(jwt_payload_groups)],
 	]
-} else = [] {
-	true
-}
+} else = []
 
 google_cloud_serverless_authentication_service_account = s {
 	s := data.google_cloud_serverless_authentication_service_account
-} else = "" {
-	true
-}
+} else = ""
 
 google_cloud_serverless_headers = h {
 	input.enable_google_cloud_serverless_authentication
 	h := get_google_cloud_serverless_headers(google_cloud_serverless_authentication_service_account, input.to_audience)
-} else = {} {
-	true
-}
+} else = {}
 
 routing_key_headers = h {
 	input.enable_routing_key
 	h := [["x-pomerium-routing-key", crypto.sha256(input.session.id)]]
-} else = [] {
-	true
-}
+} else = []
 
 pass_access_token_headers = h {
 	input.pass_access_token
 	h := [["Authorization", concat(" ", ["Bearer", session.oauth_token.access_token])]]
-} else = [] {
-	true
-}
+} else = []
 
 pass_id_token_headers = h {
 	input.pass_id_token
 	h := [["Authorization", concat(" ", ["Bearer", session.id_token.raw])]]
-} else = [] {
-	true
-}
+} else = []
+
+session_id_token = v {
+	v := session.id_token.raw
+} else = ""
```

Comment:
In rego it is necessary to use constructs like this because if `session.id_token.raw` doesn't exist, the rule will produce nothing at all and cause any rule using it to also fail in the same way. E.G. if the `id_token` didn't exist, none of the `set_request_headers` would be set.